benchmark/sirun/rewrite/server.js

```diff
@@ -0,0 +1,34 @@
+'use strict'
+
+const fs = require('fs')
+const path = require('path')
+
+const { Rewriter } = require('../../../main')
+const { port, reqs } = require('./common')
+
+const csiMethods = [
+  { src: 'concat' },
+  { src: 'join' },
+  { src: 'parse' },
+  { src: 'plusOperator', operator: true }
+]
+const filename = path.join(__dirname, 'node_modules', 'test', 'file.js')
+const contentToRewrite = fs.readFileSync(filename, 'utf8').toString()
+const orchestrion = fs.readFileSync(path.join(__dirname, 'orchestrion.yml'), 'utf8').toString()
+
+const rewriter = new Rewriter({ csiMethods, orchestrion })
+
+let connectionsMade = 0
+
+const server = require('http').createServer((req, res) => {
+  const code = contentToRewrite
+  rewriter.rewrite(code, filename, ['iast', 'orchestrion'])
```

Comment:
Similar to my comment above, I would likely run this part in a loop to make sure the process overhead is minimal and to make sure the result is stable. Right now, this shows up with a difference while the benchmark code did not change. That is an indicator for it not yet running stable :)

What I am uncertain about is if we want to also take reading the files into account or not. I guess not, since it's an overhead that we can not control?

Reply:
You're right, a loop here will reduce the noise related with the request.

> What I am uncertain about is if we want to also take reading the files into account or not. I guess not, since it's an overhead that we can not control?

When this library is used, the file content is loaded by node core, not by us, I think that it is something that we shouldn't test.